planet/api/experimental/basemaps_client.py

```diff
@@ -71,3 +81,23 @@ def list_quads_in_mosaic(
             for s in quad_list:
                 yield s
             url = resp.json().get('_links', {}).get('_next')
+
+    def get_quads_in_mosaic_for_region(self, mosaic_id, aoi):
+        url = self.base_url + f'basemaps/v1/mosaics/{mosaic_id}/quads/search?minimal=true'
+        resp = self._post(url, aoi).json()
+        return resp['items']
+
+    def get_time_series(self, aoi, start_date, end_date):
+        mosaic_series = self.list_mosaic_series(name__is='Global Monthly')['series'][0]
+        series_id = mosaic_series['id']
+        geometry = aoi[0]['config']
+        time_series = []
+        mosaics = self.list_mosaics_in_mosaic_series(series_id)['mosaics']
+        for mosaic in mosaics:
+            date = mosaic['first_acquired']
+            if start_date <= date < end_date:
+                mosaic_id = mosaic['id']
+                quads = [quad for quad in self.get_quads_in_mosaic_for_region(mosaic_id, geometry)]
+                mosaic['quads'] = quads
+                time_series.append(mosaic)
+        return time_series
```

Comment:
Nice! A great next step would be to support downloading the imagery in the timeseries into one NDArray with a time dimension